data/sql_updates/filings.sql

```diff
@@ -4,57 +4,58 @@ select
-    total_individual_contributions,
-    net_donations,
-    total_disbursements,
-    total_independent_expenditures,
-    total_communication_cost,
-    beginning_cash_on_hand as cash_on_hand_beginning_period,
-    ending_cash_on_hand as cash_on_hand_end_period,
-    debts_owed_by as debts_owed_by_committee,
-    debts_owed_to as debts_owed_to_committee,
+    ttl_receipts as total_receipts,
+    ttl_indt_contb as total_individual_contributions,
+    net_dons as net_donations,
+    ttl_disb as total_disbursements,
+    ttl_indt_exp as total_independent_expenditures,
+    ttl_communication_cost as total_communication_cost,
+    coh_bop as cash_on_hand_beginning_period,
+    coh_cop as cash_on_hand_end_period,
+    debts_owed_by_cmte as debts_owed_by_committee,
+    debts_owed_to_cmte as debts_owed_to_committee,
     -- personal funds aren't a thing anymore
-    house_personal_funds,
-    senate_personal_funds,
-    opposition_personal_funds,
-    fh.treasurer_name,
-    file_numeric as file_number,
-    previous_file_numeric as previous_file_number,
+    hse_pers_funds_amt as house_personal_funds,
+    sen_pers_funds_amt as senate_personal_funds,
+    oppos_pers_fund_amt as opposition_personal_funds,
+    filing_history.tres_nm as treasurer_name,
+    file_num as file_number,
+    prev_file_num as previous_file_number,
     report.rpt_tp_desc as report_type_full,
-    report_pgi as primary_general_indicator,
-    request_type,
-    amendment_indicator,
-    update_date,
+    rpt_pgi as primary_general_indicator,
+    request_tp as request_type,
+    amndt_ind as amendment_indicator,
+    lst_updt_dt as update_date,
     report_pdf_url_or_null(
-        begin_image_numeric,
-        report_year,
+        begin_image_num,
+        rpt_yr,
         com.committee_type,
-        fh.form_type
+        filing_history.form_tp
     ) as pdf_url,
-    means_filed(begin_image_numeric) as means_filed,
-    report_fec_url(begin_image_numeric::text, fh.file_numeric::integer) as fec_url
-from vw_filing_history fh
-left join ofec_committee_history_mv_tmp com on fh.committee_id = com.committee_id and get_cycle(fh.report_year) = com.cycle
-left join ofec_candidate_history_mv_tmp cand on fh.committee_id = cand.candidate_id and get_cycle(fh.report_year) = cand.two_year_period
-left join dimreporttype report on fh.report_type = report.rpt_tp
-where report_year >= :START_YEAR
+    means_filed(begin_image_num) as means_filed,
+    report_fec_url(begin_image_num::text, filing_history.file_num::integer) as fec_url
+from disclosure.f_rpt_or_form_sub filing_history
+left join ofec_committee_history_mv_tmp com on filing_history.cand_cmte_id = com.committee_id and get_cycle(filing_history.rpt_yr) = com.cycle
+left join ofec_candidate_history_mv_tmp cand on filing_history.cand_cmte_id = cand.candidate_id and get_cycle(filing_history.rpt_yr) = cand.two_year_period
+left join staging.ref_rpt_tp report on filing_history.rpt_tp = report.rpt_tp_cd
+where rpt_yr >= :START_YEAR
```

Comment:
Ah okay - I was noting the change above and saw the column was renamed there, but this is referring to its original name.  Note that you do need to use the original column name for things like `ORDER BY` and `GROUP BY`, so it's confusing at times. :-)